client/man/default.conf.5

```diff
@@ -263,6 +263,12 @@ system\-wide IPA client configuration file
 .TP
 .I /etc/ipa/server.conf
 system\-wide IPA server configuration file
+.TP
+.I /etc/ipa/installer.conf
+IPA configuration used while installing an IPA server or replica
+.TP
+.I /etc/ipa/cli_installer.conf
```

Comment:
When I read it as command line interface first but it is intended as client. May be we can rename it as client_installer.conf and server_installer.conf. OR may be it is just me to read like that :)

Reply:
The names of the contexts are fixed in the code. The code just loads additional configuration files named as `/etc/ipa/${context}.conf` and the context for client installer is `cli_installer`. You cannot rename that one.

Reply:
Yeah it's less than ideal but it's been in IPA since 2.0 so we're stuck with it.